packages/components/src/progress-bar/styles.ts

```diff
@@ -21,11 +21,12 @@ const animateProgressBar = keyframes( {
 // Width of the indicator for the indeterminate progress bar
 export const INDETERMINATE_TRACK_WIDTH = 50;
 
-export const Track = styled.div`
+export const Track = styled.div< { width?: number } >`
```

Comment:
Does it make sense to move this to its own `TrackProps` type and use those in the `ProgressBarProps` type as well? It could potentially help make an easier distinction of which types belong where.

Reply:
I can do it, but would it be worth it to create a whole new type for a single prop type? I think this simple and explicit.